library/src/main/java/com/digi/xbee/api/models/XBeeProtocol.java

```diff
@@ -263,6 +263,10 @@ else if (firmwareVersion.startsWith("3"))
 		case XBEE3_DM_LR_868:
 			return DIGI_MESH;
 		case S2C_P5:
+			if (firmwareVersion.toLowerCase().startsWith("b"))
```

Comment:
Is it starts with or ends with?

Reply:
Starts, current versions are A063 for Zigbee and B003 for DM. It is expected that 802 fw starts with C.